sw/device/tock/boards/opentitan/Cargo.toml

```diff
@@ -18,9 +18,5 @@ opt-level = "z"
 debug = true
 
 [dependencies]
-components = { git = "https://github.com/tock/tock" }
-rv32i = { git = "https://github.com/tock/tock" }
-capsules = { git = "https://github.com/tock/tock" }
-kernel = { git = "https://github.com/tock/tock" }
-ibex = { git = "https://github.com/tock/tock" }
-lowrisc = { git = "https://github.com/tock/tock" }
+earlgrey = { path = "../../chips/earlgrey" }
+tock_deps = { path = "../../tock_deps" }
```

Comment:
missing newline.

Reply:
Does cargo need a newline at the EOF as well?